src/test/java/pl/psnc/dei/iiif/ConversionDataHolderTransformerTest.java

```diff
@@ -0,0 +1,155 @@
+package pl.psnc.dei.iiif;
+
+import lombok.SneakyThrows;
+import org.apache.commons.io.IOUtils;
+import org.apache.jena.atlas.json.JSON;
+import org.apache.jena.atlas.json.JsonObject;
+import org.apache.jena.atlas.json.JsonValue;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.psnc.dei.model.Aggregator;
+import pl.psnc.dei.model.Record;
+import pl.psnc.dei.model.conversion.ConversionData;
+import pl.psnc.dei.model.conversion.ConversionTaskContext;
+
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringRunner.class)
+public class ConversionDataHolderTransformerTest {
+    private final String RECORD_IDENTIFIER = "/11602/HERBARWUXWUXAUSTRIAX102495";
+
+    @Value("classpath:iiif/europeana_record.json")
+    private Resource europeanaRecordResource;
+
+    @Value("classpath:iiif/europeana_record_raw.json")
+    private Resource europeanaRecordRawResource;
+
+    private JsonObject europeanaRecordJson;
+    private JsonObject europeanaRecordJsonRaw;
+
+    private JsonObject europeanaAggregatorData;
+
+    // this should be filled if integration with DDB will be implemented
+    // so far there is no way to obtain this data from programme runtime
+//    private final JsonObject ddbRecordJson = null;
+//
+//    private final JsonObject ddbAggregatorData = this.DDB_RECORD_JSON;
+
+    private ConversionDataHolderTransformer conversionDataHolderTransformer;
+
+    @Before
+    @SneakyThrows
+    public void prepareEuropeanaData() {
+        String europeanaRecordResourceString =
+                String.join("",
+                        IOUtils.readLines(
+                                this.europeanaRecordResource.getInputStream(),
+                                StandardCharsets.UTF_8
+                        )
+                );
+        this.europeanaRecordJson = JSON.parse(europeanaRecordResourceString);
```

Comment:
`JSON#parse` accepts `InputStream` as a parameter, so you can simplify all the lines above just to `this.europeanaRecordJson = JSON.parse(this.europeanaRecordResource.getInputStream());`